clients/tests/accuracy_test_2D.cpp

```diff
@@ -58,12 +58,15 @@ class accuracy_test_complex_2D_pow2_double : public ::testing::Test
     virtual void TearDown() {}
 };
 // 65536=pow(2,16)                                 //8388608 = pow(2,23)
-#define POW2_RANGE                                        \
-    {2, 4}, {8, 16}, {32, 128}, {256, 512}, {1024, 2048}, \
-    {                                                     \
-        4096, 8192                                        \
-    } /* malloc fail on 4GB Fiji Nano on the following size \                    \
-, {16384, 32768}, {65536, 131072}, {262144, 524288} */
+#define POW2_RANGE                          \
```

Comment:
POW2_RANGE seems to be shared for regular C2C cases also. The better way could be: define a sub-range just for R2C cases.

Reply:
Sounds reasonable; done.